sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -134,3 +138,25 @@ def _eventgrid_data_typecheck(event):
             "Data in EventGridEvent cannot be bytes. Please refer to"
             "https://docs.microsoft.com/en-us/azure/event-grid/event-schema"
         )
+
+def _cloud_event_to_generated(cloud_event, **kwargs):
```

Comment:
I'm on the fence to put it in the _helpers file  vs leaving this as a standalone method in _models.py - any strong opinions here? 

Nevertheless, it's an implementation detail and not blocking

Reply:
don't care  :p